charts/kaito/workspace/Chart.yaml

```
@@ -37,3 +37,7 @@ dependencies:
   version: "2.16.3"
   repository: https://fluxcd-community.github.io/helm-charts
   condition: featureGates.gatewayAPIInferenceExtension
+- name: gpu-feature-discovery
+  version: 0.17.4
```

New version was released recently

```suggestion
  version: 0.18.0
```